core/domain/stats_services.py

```diff
@@ -295,6 +295,28 @@ def save_stats_model_transactional(exploration_stats):
         _save_stats_model, exploration_stats)
 
 
+def get_view_multi(exploration_ids, version_numbers):
+    """Retrieves the view counts for the given exploration IDs.
+
+        Args:
+            exploration_ids: list(str). The list of exploration IDs to get view
+                counts for.
+            version_numbers: list(int). List of version numbers of explorations.
+
+        Returns:
+            list(int). The number of times each exploration in exploration_ids
+                has been viewed.
+    """
+    exploration_stats_models = (
+        stats_models.ExplorationStatsModel.get_multi_stats_models(
+            exploration_ids, version_numbers))
+
+    return [
+        exploration_stats_models[i].num_starts_v1 if (
```

Comment:
Why only v1? Also in general shouldn't we get the domain objects and extract the total num starts property?

Reply:
Yep. Done.